src/mgr/Mgr.cc

```diff
@@ -649,6 +648,7 @@ void Mgr::handle_fs_map(ref_t<MFSMap> m)
           {}, &c->outbl, &c->outs, c);
     }
   }
+  fs_map_cond.notify_all();
```

Comment:
this has been moved from the start of the routine to the end -- what does it fix?

Reply:
@vshankar I had thought of moving the notification to the last since my patch was working; but my python code wasn't what it was supposed to be; I'll be reverting back these changes